bigquery_etl/query_scheduling/templates/public_data_json_airflow_dag.j2

```diff
@@ -40,6 +40,7 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
         task_id='wait_for_{{ dependency.task_name }}',
         external_dag_id='{{ dependency.dag_name }}',
         external_task_id='{{ dependency.task_name }}',
+        check_existence=True,
```

Comment:
Added this check to make sure tasks fail if referenced tasks/DAGs do not exist instead of poking for a long time.